### ISSUE TITLE
minimal Dockerfile example yields broken notebook (404: not found) on mybinder.org 

### ISSUE THREAD

**Issue by N-Coder** [jupyterhub/binder#240]
Opening any of the two binder links of the [minimal Dockerfile example](https://github.com/binder-examples/minimal-dockerfile) yields a notebook that simply displays a 404 error page:
![image](https://user-images.githubusercontent.com/1306276/133405961-0def33e5-f8d9-4b85-90e5-3e788ceac679.png)
[![Binder](https://mybinder.org/badge.svg) master branch](https://mybinder.org/v2/gh/binder-examples/minimal-dockerfile/master)
[![Binder](https://mybinder.org/badge.svg) truly-minimal branch](https://mybinder.org/v2/gh/binder-examples/minimal-dockerfile/truly-minimal)
Running `jupyter-repo2docker https://github.com/binder-examples/minimal-dockerfile` locally yields a working notebook.

**Comment by manics** [comment]
That'll be due to the default URL changing to jupyterlab https://github.com/jupyterhub/binderhub/pull/1354

Options are to either change the binder badge to pass `/tree` as the starting URL, or to install JupyterLab in the minimal Dockerfile.

**Comment by N-Coder** [comment]
I also get the same problem when pasting the link to the GitHub repo on the mybinder.org starting page, so it's probably all links that are generated there: https://mybinder.org/v2/gh/binder-examples/minimal-dockerfile/HEAD
![image](https://user-images.githubusercontent.com/1306276/133410585-b178e16c-606b-4320-9421-83b636d14641.png)

**Comment by betatim** [comment]
mybinder.org will try to send you to the URL at which JupyterLab is, but because the minimal example doesn't install JupyterLab you get a 404. This means that the minimal example needs updating so that it installs Jupyter Lab. Do you want to tackle that in a PR to https://github.com/binder-examples/minimal-dockerfile?

**Comment by N-Coder** [comment]
So that basically means that Docker Containers without JupyterLab are no longer really supported on mybinder.org?
I guess then there are also some parts of the documentation that should be updated to reflect this, especially [here](https://mybinder.readthedocs.io/en/latest/tutorials/dockerfile.html#preparing-your-dockerfile) and [here](https://mybinder.readthedocs.io/en/latest/examples/sample_repos.html#minimal-dockerfiles-for-binder).
One could either remove the part about only installing `notebook` or make the information about how to [switch between both versions](https://mybinder.readthedocs.io/en/latest/howto/user_interface.html#jupyterlab) more promininent with a warning on both pages saying that you'll need to manually fix your URLs.

**Comment by betatim** [comment]
> So that basically means that Docker Containers without JupyterLab are no longer really supported on mybinder.org?

Not quite. You can make a container that doesn't have lab installed but then you will have to specify the URL that users should be redirected to because the default URL is the one JupyterLab uses.

Yes, there are unfortunately a lot of places in the docs that need updating that haven't yet been updated.

**Comment by N-Coder** [comment]
I opened PRs in #243, binder-examples/minimal-dockerfile#4 and binder-examples/minimal-dockerfile#5, feel free to edit them as you deem suitable. I also didn't update the [example overview page](https://github.com/jupyterhub/binder/blob/master/doc/examples/sample_repos.md), as that probably needs the other PRs merged and I currently have no environment at hand to actually run the script.
